fck_aws_sso/authorize_sso.py

```diff
@@ -37,15 +37,16 @@ def authorize_sso(url, code, headless=True, user_data_dir=None):
 
         logging.debug("Waiting for the allow page to load")
         login_button = WebDriverWait(driver, 1000).until(
-            EC.element_to_be_clickable((By.ID, "cli_login_button"))
+            EC.element_to_be_clickable((By.CSS_SELECTOR, ".awsui_button_vjswe_1nbam_105.awsui_variant-primary_vjswe_1nbam_248"))
```

Comment:
hmm this css selector feels like something that is dynamically generated by some library (`_vjswe_1nbam_105` part) so I suspect it might break soon, maybe it is possible to make it more robust? maybe we could select that by button text, or xpath?